src/Mvc/Mvc.Core/src/MvcCoreLoggerExtensions.cs

```diff
@@ -155,6 +158,16 @@ internal static class MvcCoreLoggerExtensions
 
         static MvcCoreLoggerExtensions()
         {
+            _controllerFactoryExecuting = LoggerMessage.Define<string, string>(
+                LogLevel.Debug,
+                new EventId(1, "ControllerFactoryExecuting"),
+                "Executing controller factory for controller {Controller} ({AssemblyName})");
+
+            _controllerFactoryExecuted = LoggerMessage.Define<string, string>(
+                LogLevel.Debug,
+                new EventId(2, "ControllerFactoryExecuted"),
+                "Executed controller factory for controller {Controller} ({AssemblyName})");
```

Comment:
re: your question about event ids, they need to be unique per logging source, which these are :+1: